### PR TITLE
Release/v0.7.5

### PR DESCRIPTION
**Pull request by cbgbt**
**Description of changes:**

Updated SSM Agent to **3.2.1705.0**.

Bumped version

**Testing done:**

- [x] Launched `aws-k8s-1.27` ami with new control container set in userdata.
- [x] Verified SSM agent is reporting as 3.2.1705.0.

**Terms of contribution:**

By submitting this pull request, I agree that this contribution is dual-licensed under the terms of both the Apache License, version 2.0, and the MIT license.

### CODE REVIEW

**Review by yeazelm — APPROVED** [review]
LGTM!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 0.7.5
+
+* Update SSM agent to 3.2.1705.0 ([#51])
+
+[#50]: https://github.com/bottlerocket-os/bottlerocket-control-container/pull/51
```
Comment (stmcginnis) [inline CHANGELOG.md]: There's a typo here, with the label being `#50` but the reference above and actual URL being for PR 51.

**Review by stmcginnis — CHANGES_REQUESTED** [review]

**Comment by cbgbt** [comment]
Fixed the typo pointed out by @stmcginnis

**Review by stmcginnis — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (cbgbt) [reply CHANGELOG.md]: Nice catch!

**Pull request merged**